components/notebook-controller/Makefile

```diff
@@ -77,12 +80,17 @@ run: manifests generate fmt vet ## Run a controller from your host.
 
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
+=======
```

Comment:
I think there's some conflict resolution that needs to happen in this file

Reply:
Done!